PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -795,32 +819,46 @@ def getMETUncertainties(self, process, metType, metModName, electronCollection,
             #electron projection ==
             pfCandsNoJetsNoEle = cms.EDProducer("CandPtrProjector", 
                                                 src = cms.InputTag("pfCandsNoJets"+postfix),
-                                                veto = electronCollection
+                                                veto = electronCollection,
+                                                useDeltaRforFootprint = cms.bool(False),
                                                 )
+            if not self._parameters["onMiniAOD"].value:
```

Comment:
IIUC, ```if not onMiniAOD:``` should work

Reply:
onMiniAOD not defined in this function

Reply:
ok, it looks like I thought more of the ConfigToolBase than it really is.
```getvalue('onMiniAOD')``` should exist though and do the same as ```self._parameters["onMiniAOD"].value```
https://github.com/cms-sw/cmssw/blob/4546ca54b1ef88845cbc585619642f4b9e16ed83/PhysicsTools/PatAlgos/python/tools/ConfigToolBase.py#L90-L93